src/tests/EdFi.DataManagementService.Tests.E2E/Features/SchoolYearReferenceValidation.feature

```diff
@@ -0,0 +1,138 @@
+                  | 2024       | true              | School Year 2024      |
+                  | 2029       | false             | School Year 2029      |
+              Then all responses should be 201 or 200
+
+        Scenario: Try creating a resource using a valid school year
+             When a POST request is made to "/ed-fi/calendars" with
+                  """
+                  {
+                      "calendarCode": "321",
+                      "schoolReference": {
+                        "schoolId": 535
+                      },
+                      "schoolYearTypeReference": {
+                        "schoolYear": 2024
+                      },
+                      "calendarTypeDescriptor": "uri://ed-fi.org/CalendarTypeDescriptor#Student Specific",
+                      "gradeLevels": []
+                  }
+                  """
+             Then it should respond with 201 or 200
+
+        @Ignore
+        Scenario: Try creating a resource using an invalid school year
+             When a POST request is made to "/ed-fi/calendars" with
+                   """
+                  {
+                      "calendarCode": "325",
+                      "schoolReference": {
+                        "schoolId": 535
+                      },
+                      "schoolYearTypeReference": {
+                        "schoolYear": 2099
+                      },
+                      "calendarTypeDescriptor": "uri://ed-fi.org/CalendarTypeDescriptor#Student Specific",
+                      "gradeLevels": []
+                  }
+                  """
+             Then it should respond with 409
+              And the response body is
+                  """
+                     {
+                      "detail": "The referenced 'SchoolYearType' item does not exist.",
+                      "type": "urn:ed-fi:api:data-conflict:unresolved-reference",
+                      "title": "Unresolved Reference",
+                      "status": 409,
+                      "correlationId": null
+                     }
+                  """
+
+        #CalendarDates / School Year
+        @Ignore
+        Scenario: Try creating a CalendarDate using a valid Calendar reference
+              
+              When a POST request is made to "/ed-fi/calendars" with
+                  """
+                  {
+                      "calendarCode": "451",
+                      "schoolReference": {
+                        "schoolId": 535
+                      },
+                      "schoolYearTypeReference": {
+                        "schoolYear": 2029
+                      },
+                      "calendarTypeDescriptor": "uri://ed-fi.org/CalendarTypeDescriptor#Student Specific",
+                      "gradeLevels": []
+                  }
+                  """
+             Then it should respond with 201 or 200
+             When a POST request is made to "/ed-fi/calendarDates" with
```

Comment:
Each scenario should have a single `When` statement. Is the first `When` trying to setup initial date?

Big picture, the scenario could be more descriptive of the _purpose_ for this scenario. We can read to see _what_ it is doing, but we need to understand _why_.